python/lsst/daf/butler/core/dataUnit.py

```diff
@@ -343,16 +404,19 @@ def _initDataUnitJoins(self, config, builder):
         """
         for dataUnitJoinName, dataUnitJoinDescription in config.items():
             table = None
+            isView = None
             if 'tables' in dataUnitJoinDescription and builder is not None:
                 for tableName, tableDescription in dataUnitJoinDescription['tables'].items():
                     table = builder.addTable(tableName, tableDescription)
+                    isView = "sql" in tableDescription
```

Comment:
Could "sql" in YAML potentially mean something else than view?

Reply:
In general, yes (see `ExposureRangeJoin`), inside a `tables` section it always means a view.